opengrok-indexer/src/main/java/org/opengrok/indexer/index/IndexDatabase.java

```diff
@@ -495,7 +495,11 @@ public void update() throws IOException {
 
                     IndexDownArgs args = new IndexDownArgs();
                     Statistics elapsed = new Statistics();
-                    LOGGER.log(Level.INFO, "Starting traversal of directory {0}", dir);
+                    if (dir.isEmpty()) {
+                        LOGGER.log(Level.INFO, "Starting traversal of directory 'source root directory'");
+                    } else {
+                        LOGGER.log(Level.INFO, "Starting traversal of directory " + "'" + "{0}" + "'", dir);
```

Comment:
how does the output produced by this statement look in the indexer log now ?

Reply:
the directory name  is wrapped in single quotes now, as suggested.

Reply:
No, it's not. `'` has a special meaning and now it would print `'{0}'` instead of the directory name. That's why double quotes `''` are required.

Reply:
That's why I suggested to actually run the indexer. You need to see the changed code in action, otherwise this kind of blind PR does not make sense I'd say.